frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -31,26 +36,40 @@ const props = defineProps({
         type: Array as PropType<FamApplicationUserRoleAssignmentGet[]>,
         default: [],
     },
+    applicationAdmins: {
+        type: Array as PropType<FamAppAdminGet[]>,
+        default: [],
+    },
 });
 
 const userRoleAssignments = shallowRef<FamApplicationUserRoleAssignmentGet[]>(
     props.userRoleAssignments
 );
 
+const applicationAdmins = shallowRef<
+    FamAppAdminGet[]
+>(props.applicationAdmins);
+
 onUnmounted(() => {
     resetNotification();
 });
 
 const onApplicationSelected = async (e: DropdownChangeEvent) => {
     setSelectedApplication(e.value ? JSON.stringify(e.value) : null);
-    userRoleAssignments.value = await fetchUserRoleAssignments(
-        selectedApplication.value?.application_id
-    );
+    if (e.value.application_id === FAM_APPLICATION_ID) {
+        applicationAdmins.value = await fetchApplicationAdmins(
+            FAM_APPLICATION_ID
```

Comment:
As @MCatherine1994 mentioned in the comment in `fetchData` on function `fetchApplicationAdmins()`, passing this `FAM_APPLICATION_ID` is wrong but will need to wait for new backend api to be ready.